src/arch/xtensa/gdb/Makefile.am

```diff
@@ -0,0 +1,19 @@
+noinst_LIBRARIES = libgdb.a
```

Comment:
I guess it's part of something bigger and more changes are incoming, because it won't be included in final binary until you add `SUBDIRS+=gdb` in parrent Makefile and path to that makfile in configure.ac

Reply:
Yes, it will be linked in next commits as for now I want to just commit important sources.